clap_builder/src/builder/command.rs

```diff
@@ -3678,6 +3678,12 @@ impl Command {
         self.get_arguments().filter(|a| a.is_positional())
     }
 
+    /// Returns `true` if this `Command` has any positional arguments.
+    #[inline]
+    pub fn has_positionals(&self) -> bool {
```

Comment:
We've generally not been exposing helper methods like this and would prefer not to start doing so

Reply:
How about I `doc(hidden)` it? 
There is [`Command::has_subcommands`][1] already. So I thought I could make `has_positionals` public.

[1]: https://docs.rs/clap/latest/clap/struct.Command.html#method.has_subcommands

Reply:
has subcommands is a bit more black and white compared to the different states arguments can be in.

Doc hidden would be fine if this were being used by a macro.  Being used here means we still have compatibility guarantees around it.